float64 timestamp

float64 info_battery
int32 drone_mode
